book/getting-started/docker-setup.md

```diff
@@ -0,0 +1,55 @@
+# Setting Up Your Environment for Wallaroo in Docker
+
+To get you up and running quickly with Wallaroo, we've provided a Docker image which includes Wallaroo and related tools needed to run and modify a few example applications. We should warn that this Docker image was created with the intent of getting users started quickly with Wallaroo and is not intended to be a fully customizable development environment or suitable for production.
+
+## Installing Docker
+
+### MacOS
+
+There are [instructions](https://docs.docker.com/docker-for-mac/) for getting Docker up and running on MacOS on the [Docker website](https://docs.docker.com/docker-for-mac/).  We recommend the 'Standard' version of the 'Docker for Mac' package.
+
+Installing Docker will result in it running on your machine. After you reboot your machine, that will no longer be the case. In the future, you'll need to have Docker running in order to use a variety of commands in this book. We suggest that you [set up Docker to boot automatically](https://docs.docker.com/docker-for-mac/#general).
```

Comment:
"At the end of the installation, the installer starts the docker daemon. After rebooting, however, the daemon is not started automatically.  If you want to have docker start automatically, we suggest [these instructions](...). If you prefer to leave it off by default, you will need to manually start the docker daemon before using it, with the following command ..."

Reply:
same as above